python/cudf_polars/cudf_polars/containers/dataframe.py

```diff
@@ -295,7 +295,7 @@ def filter(self, mask: Column) -> Self:
         table = plc.stream_compaction.apply_boolean_mask(self.table, mask.obj)
         return type(self).from_table(table, self.column_names).sorted_like(self)
 
-    def slice(self, zlice: tuple[int, int] | None) -> Self:
+    def slice(self, zlice: tuple[int, int | None] | None) -> Self:
```

Comment:
*question*: When can the `length` parameter be `None`?

Reply:
When its unspecified eg. `pl.col("a").slice(1)`